tooling/main.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+
+const (
+	// defaultKubebuilderPath is the fixed value defined in the controller-runtime to looking for the
+	// binaries.
+	defaultKubebuilderPath = "/usr/local/kubebuilder/bin"
+
+	// kubernetesVersion defines the version used to download the binaries
+	kubernetesVersion = "1.16.4"
```

Comment:
maybe want to consider moving this to a separate branch so that it's easy to bump the tooling (can maybe just put this code on the existing tooling branch)

Reply:
Done.